napalm_ios/ios.py

```diff
@@ -592,6 +592,13 @@ def get_optics(self):
             port_detail['physical_channels'] = {}
             port_detail['physical_channels']['channel'] = []
 
+            # If interface is shutdown it returns "N/A" as output power.
+            # Converting that to -40.0 float
+            try:
+                float(output_power)
+            except ValueError:
+                output_power = -40.0
```

Comment:
Out of curiosity, why -40? Is that an insane value that clearly indicates it is not supported/available? I am not an optics person myself :P

In any case, I am fine with having such a value if it makes sense but we should make it clear here:

https://github.com/napalm-automation/napalm-base/blob/develop/napalm_base/base.py#L1332